### DISCUSSION TITLE
Storage signed URLs still requiring authorization header

### DISCUSSION THREAD

**Discussion by zhadavi** [supabase/supabase#20720] [Questions]
I'm uploading a file using Ant Design's [`Upload` component](https://ant.design/components/upload). I'm getting generating a signed upload URL via the following code
```javascript
<Upload action={async (file) => {
   const { data, error } = await supabaseClient.storage
              .from("documents")
              .createSignedUploadUrl(`${task.id}/${file.name}`);
  return data.signedUrl;
}} />
```
However when I upload to this URL I get a 400 and see this error in the logs:
```
headers must have required property 'authorization'
```
to my understanding, this is the whole point of using a signed URL: to not require this header anymore. Does anyone know what could be going wrong here? Am I using this API incorrectly?

I've checked RLS and I have a policy that any authenticated user is allowed to upload to the `documents` bucket.

**Comment by RonyTrespalacios** [comment]
Hello, Have you solved it? I got the same issue.

**Comment by nyarthan** [comment]
The default method of the Ant Design Upload component is `POST`. If you change it to `PUT` it should work.
